frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -368,6 +379,7 @@ class DashboardHeader extends Component {
       buttons.push(
         <Tooltip key="edit-dashboard" tooltip={t`Edit dashboard`}>
           <DashboardHeaderButton
+            visibleOnSmallScreen={false}
```

Comment:
It seems to be the only place where `DashboardHeaderButton` is used, we could simply assume this flag is false

Reply:
@uladzimirdev do I miss something?

Reply:
I didn't get, sorry @ranquild could you please express your idea?

I see plenty of places where `DashboardHeaderButton` is used on the screenshot, we specify that using default props that buttons are visible on small screens, except this specific case, when we have to override the default value with false. Do you mean to use the opposite prop?

Reply:
👍